charts/pulsar/templates/NOTES.txt

```diff
@@ -0,0 +1,18 @@
+Thank you for installing Apache Pulsar Helm chart version {{ .Chart.Version }}.
+
+!! WARNING !!
+
+Important Security Disclaimer for Apache Pulsar Helm Chart Usage:
+
+This Helm chart is provided with a default configuration that does not
+meet the security requirements for production environments or sensitive
+data handling. Users are strongly advised to thoroughly review and
+customize the security settings to ensure a secure deployment that
```

Comment:
Maybe indicate what security aspects in particular e.g. encryption, authentication, authorization? etc.

Reply:
We don't currently have documentation about all parts to cover. We can update this after we have proper guidance.